app/connectors/TaxReturnsConnector.scala

```diff
@@ -50,11 +50,22 @@ class TaxReturnsConnector @Inject()(
       }
   }
 
-  def submit(payload: TaxReturn)(implicit hc: HeaderCarrier): Future[Either[ServiceError, Option[String]]] = {
+  def getCalculation(pptReference: String)(implicit hc: HeaderCarrier): Future[Either[ServiceError, Calculations]] = {
+    val url = appConfig.pptReturnCalculationUrl(pptReference)
```

Comment:
How do we know which return / period this is for?

Reply:
its for what ever is in the user answers (cache), its a "show me the calculations of what i may be about to submit" so that the frontend isnt assuming it does the calculations correctly

Reply:
I guess in answer to the original question the calculation doesnt care which return its for, its agnostic. This is where consideration would have to be made if the tax amount ever did change, however atleast all that consideration is now in one place